backend/dataall/modules/s3_datasets/services/dataset_service.py

```diff
@@ -1,6 +1,7 @@
 import os
 import json
 import logging
+from sqlalchemy import inspect
```

Comment:
unused

I usually do a `Code -> Optimize Imports` before commit to automatically remove unused imports